conf/db/migration/metrics_portal_ddl/common/V3__create_alerts_table.sql

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2015 Groupon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TABLE IF NOT EXISTS portal.alerts (
```

Comment:
Yes, tested against local pg db. And removed the "IF NOT EXISTS". I put that in because the tests seemed to be interfering but I fixed the tests and missed removing here